src/components/edit/EditServicesTable.tsx

```diff
@@ -175,7 +214,14 @@ export const EditServicesTable: React.FC = () => {
         rowModesModel={rowModesModel}
         onRowModesModelChange={(newMode) => setRowModesModel(newMode)}
         onRowEditStop={handleRowEditStop}
-        onProcessRowUpdateError={(error) => console.log(error)}
+        onProcessRowUpdateError={(error) => {
+          console.error("Row update error:", error);
+          if (error instanceof Error) {
```

Comment:
How does this get triggered?

Reply:
The way you guys have it set up now - 
```
const processRowUpdate = async (newRow: ServiceRow) => {
    try {
      try all the things
    } catch (error) {
      catch all the errors
    }
```
you are already catching any errors in processRowUpdate, so this onProcessRowUpdateError code won't run! But once you revert processRowUpdate to the original code (see my other comment), this should be good.